### ISSUE TITLE
No icons with Nerd Fonts

### ISSUE THREAD

**Issue by MerrinX** [Powerlevel9k/powerlevel9k#1391]
#### Describe Your Issue
What is happening?
No icons or glyphs on my powerlevel9k

#### Have you tried to debug or fix it?
Have tried different fonts, using urxvt terminal. Tried downloading nerd fonts and installed again, to see if the icons set maybe was bugged. 

Have you tinkered with your settings, and what happened when you did? Did you find a bit of code that you think might be the culprit? Let us know what you've done so far!

#### Environment Information

This information will help us understand your configuration.

  - What version of ZSH are you using? You can use `zsh --version` to see this.5.6.2
  - Do you use a ZSH framework (e.g., Oh-My-ZSH, Antigen)? Yes i use oh-my-ZSH
  - How did you install P9k (cloning the repo, by tarball, a package from your OS, etc.,)? cloning
  - What version of P9k are you using? latest
  - Which terminal emulator do you use? urxvt

#### Issues with Fonts & Icons
You may delete this section if your issue is not font / icon related.

  - Which font do you use? Hack Bold Nerd font but i have also tried different ones
  - Which [font configuration mode] (https://github.com/bhilburn/powerlevel9k/wiki/About-Fonts) are you using? You can check this with (`echo $POWERLEVEL9K_MODE`).
  - Please share the contents of `$P9k/debug/font-issues.zsh`.
  - If this is an icon problem, does the output of `$ get_icon_names` look correct?

When doing a $get_icon_names ther is no icons

**Comment by romkatv** [comment]
Does the output of `xrdb -query` indicate that urxvt is using a Nerd Font? If not, see https://wiki.archlinux.org/index.php/Rxvt-unicode#Configuration for how to configure fonts (among other things) for urxvt.

**Comment by MerrinX** [comment]
Thank you for replying, here is my code for fonts.

`URxvt.font: xft: Hack\ Bold\ Nerd\ Font:pixelsize=11:antialias=true:hinting=true`

when doing xrdb -query, i get no output.

**Comment by romkatv** [comment]
> Thank you for replying, here is my code for fonts.

Is urxvt actually using Hack Bold Nerd? If you change this font to something else, do you notice any difference in the way text looks in urxvt?

**Comment by MerrinX** [comment]
Yes, i tried changing to Droid Sans and it changes. And the font look identical to how the Hack Bold is supposed to look. strange...

I have a feeling there is a user error on my behalf here..

**Comment by romkatv** [comment]
> I have a feeling there is a user error on my behalf here..

Almost certainly. You would make it easier for others to help you if you could answer all questions from the issue template without skipping any. I'll highlight a few sections you've skipped over that are crucial for understanding your issue.

> Most issues are best explained with a screenshot. Please share one if you can!

This is the most important thing. Without a screenshot it's anyone's guess what "no icons or glyphs" means.

> Which [font configuration mode](https://github.com/bhilburn/powerlevel9k/wiki/About-Fonts) are you using? You can check this with (`echo $POWERLEVEL9K_MODE`).

Do run the command and post its output.

Better yet, run `typeset -pm 'POWERLEVEL9K_*'`.

> Please share the contents of `$P9k/debug/font-issues.zsh`.

Here `$P9k` is a placeholder for the directory where you've cloned powerlevel9k.

> If this is an icon problem, does the output of `$ get_icon_names` look correct?

Ignore the dollar. The command you need to run is `get_icon_names`. Attach a screenshot of the output.

P.S.

If you don't mind my asking, how did you learn about the existence of powerlevel9k?

**Comment by MerrinX** [comment]
Content of font-issues.zsh
http://dpaste.com/0VTT932

![snap2020-01-01_19-25-35](https://user-images.githubusercontent.com/42249650/71644791-afa60680-2cce-11ea-8957-e23785357140.png)

![snap2020-01-01_19-26-09](https://user-images.githubusercontent.com/42249650/71644797-b765ab00-2cce-11ea-8f57-c27532466ebe.png)

**Comment by romkatv** [comment]
Thanks for posting the screenshot. Your terminal is not using Hack. See [here](https://source-foundry.github.io/Hack/font-specimen.html) for how Hack looks like. What you see in the terminal is likely a fallback that urxvt uses when the configured font is invalid. It's basically "no font" mode.

**Comment by MerrinX** [comment]
http://dpaste.com/1B30SP4

There is my .Xdefaults

I do not see how it wont pull in the fonts?
Please advice if you can

**Comment by romkatv** [comment]
Here's the question that you need to ask: "How to configure urxvt to use Hack Nerd Font?" I don't know the answer as I use neither urxvt nor Hack Nerd Font. There must be forums that are better suited to questions of this sort than an issue tracker of an unrelated abandoned project.

**Comment by MerrinX** [comment]
did a fc-list | grep Hack.. and found it to be my bad typing.
solve the issue, so can close this.

the output should be:
`URxvt.font: xft:Hack Nerd Font:pixelsize=12`

Than you for your help, much appreciated.

**Comment by romkatv** [comment]
You are most welcome.

> If you don't mind my asking, how did you learn about the existence of powerlevel9k?

Do you mind? It would allow me to help future potential users so that they don't fall into the same trap.

**Comment by MerrinX** [comment]
Watched some videos on youtube where others where tweaking their terminals, and had to do some searches, and found you from there.

**Comment by romkatv** [comment]
> Watched some videos on youtube where others where tweaking their terminals, and had to do some searches

Got it. Thanks.

> and found you from there.

I'm neither a developer nor a user of this project.
